analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -401,6 +401,7 @@ internal sealed partial class KnownType
         internal static readonly KnownType System_Security_Cryptography_DSACryptoServiceProvider = new("System.Security.Cryptography.DSACryptoServiceProvider");
         internal static readonly KnownType System_Security_Cryptography_ECDiffieHellman = new("System.Security.Cryptography.ECDiffieHellman");
         internal static readonly KnownType System_Security_Cryptography_ECDsa = new("System.Security.Cryptography.ECDsa");
+        internal static readonly KnownType System_Security_Cryptography_ECAlgorythm = new("System.Security.Cryptography.ECAlgorithm");
```

Comment:
ECAlgorithm is a common base type for ECDsa and ECDiffieHellman introduced in .Net 7 (link to the [proposal](https://github.com/dotnet/runtime/issues/61516) and the related [PR](https://github.com/dotnet/runtime/pull/61753))